Kickstarter-iOS/Views/Cells/DashboardReferrersCell.swift

```diff
@@ -42,6 +42,8 @@ internal final class DashboardReferrersCell: UITableViewCell, ValueCell {
   internal override func awakeFromNib() {
     super.awakeFromNib()
 
+    self.viewModel.inputs.awakeFromNib()
```

Comment:
I think it's best to keep this at the bottom of the function so that we know it's full executed before calling the viewModel, unless it's necessary for functionality.